cmd/app_action.go

```diff
@@ -25,9 +26,14 @@ func AppAction(c *cli.Context, providers app.Providers, fn AppActionFn) error {
 	}
 	appName := c.Args().Get(0)
 	envName := c.Args().Get(1)
+	stackName := c.String("stack-name")
```

Comment:
Does this grab the actual stack name from the command?

Reply:
Yep, `""` if the user doesn't specify.